docs/src/main/asciidoc/secret-management.adoc

```diff
@@ -112,9 +112,10 @@ If key exists in multiple PropertySources, which will take effect is decided by
 |===
 
 - Please refer to link:index.html#authorize-access-with-azure-active-directory[Authorize access with Azure AD] to make sure the link:https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals#service-principal-object[security principal] has been granted the sufficient permission to access the Azure Key Vault Secrets.
-- If common properties like `client`, `credential`, `profile`, `proxy`, `retry` are not configured in `spring.cloud.azure.keyvault.secret.property-sources[].xxx`, `spring.cloud.azure.xxx` will be used. Please refer to link:index.html#configuration[Configuration] to get more information about these common properties.
+- If common properties like `client`, `credential`, `profile`, `proxy`, `retry` aren't configured in `spring.cloud.azure.keyvault.secret.property-sources[].xxx`, `spring.cloud.azure.xxx` will be used. Please refer to link:index.html#configuration[Configuration] to get more information about these common properties.
```

Comment:
line:104 is missed
<img width="605" alt="image" src="https://user-images.githubusercontent.com/4465723/160324228-a87a67d9-d23b-456e-a896-48b8c6d26d5a.png">